nntrainer/layers/layer_node.cpp

```diff
@@ -325,6 +343,15 @@ void LayerNode::setInputLayers(const std::vector<std::string> &layers) {
                  });
 }
 
+void LayerNode::setOutputLayers(const std::vector<std::string> &layers) {
+  output_layers->clear();
+  output_layers->reserve(layers.size());
+  std::transform(layers.begin(), layers.end(),
+                 std::back_inserter(*output_layers), [](const std::string &id) {
+                   return props::Connection{id, 0};
```

Comment:
So far, we only support the one output tensor per connection. Let's leave a comments as an TODO support multi tensor outputs per connection.

Reply:
Yes, I'll add a comment with https://github.com/nnstreamer/nntrainer/pull/1732#discussion_r758029891 :)